api/opencl_platform_layer.asciidoc

```diff
@@ -1216,24 +1216,24 @@ include::{generated}/api/version-notes/CL_DEVICE_PREFERRED_INTEROP_USER_SYNC.asc
 | {CL_DEVICE_PARENT_DEVICE_anchor}
 
 include::{generated}/api/version-notes/CL_DEVICE_PARENT_DEVICE.asciidoc[]
-  | cl_device_id
-      | Returns the cl_device_id of the parent device to which this
+  | {cl_device_id}
+      | Returns the {cl_device_id} of the parent device to which this
         sub-device belongs.
         If _device_ is a root-level device, a `NULL` value is returned.
 | {CL_DEVICE_PARTITION_MAX_SUB_DEVICES_anchor}
 
 include::{generated}/api/version-notes/CL_DEVICE_PARTITION_MAX_SUB_DEVICES.asciidoc[]
-  | cl_uint
+  | {cl_uint}
       | Returns the maximum number of sub-devices that can be created when a
         device is partitioned.
 
         The value returned cannot exceed {CL_DEVICE_MAX_COMPUTE_UNITS}.
 | {CL_DEVICE_PARTITION_PROPERTIES_anchor}
 
 include::{generated}/api/version-notes/CL_DEVICE_PARTITION_PROPERTIES.asciidoc[]
-  | cl_device_partition_ property[]
+  | {cl_device_partition_property}[]
       | Returns the list of partition types supported by _device_.
-        This is an array of cl_device_partition_property values drawn from
+        This is an array of {cl_device_partition_property} values drawn from
         the following list:
 
         {CL_DEVICE_PARTITION_EQUALLY} +
```

Comment:
I can't comment on the actual line, but a few lines below here `cl_device_affinity_domain` has been missed, perhaps due to a surprising space.

Reply:
Thanks - fixed!